src/plugins/proxyconfig_stem.py

```diff
@@ -29,14 +32,14 @@ def __call__(self, line):
             # Plugin's debug or unexpected log line from tor
             self._logger.debug(line)
         else:
-            self._logger.log(self._levels.get(level, 10), '(tor)' + line)
+            self._logger.log(self._levels.get(level, 10), '(tor)' + line)    # pylint: disable=logging-not-lazy
```

Comment:
This can probably be fixed properly, like

`self._logger.log(self._levels.get(level, 10), '(tor) %s', line)`

Reply:
Okay. Thanks

Reply:
@PeterSurda thank you, now I see it.